README.md

```diff
@@ -18,11 +18,15 @@ Create `.env` file in project root with the following parameters:
 DEPLOYMENT_ACCOUNT_PRIVATE_KEY=67..14
 # Extra gas added to the estimated gas of a particular deployment/configuration transaction
 DEPLOYMENT_GAS_LIMIT_EXTRA=0.2
-# The "gasPrice" parameter set in every deployment/configuration transaction on Foreign network (in Wei).
-FOREIGN_DEPLOYMENT_GAS_PRICE=1000000000
+# The RPC channel to a Home node able to handle deployment/configuration transactions.
+HOME_RPC_URL=https://sokol.poa.network
+# The "gasPrice" parameter set in every deployment/configuration transaction on Home network (in Wei).
+HOME_DEPLOYMENT_GAS_PRICE=1000000000
 # The RPC channel to a Foreign node able to handle deployment/configuration transactions.
 FOREIGN_RPC_URL=https://sokol.poa.network
```

Comment:
Please consider the case of the deployment when you need to deploy only the NFT contract on Home chain

Reply:
Added, please check https://github.com/poanetwork/cryptokitties-xdai-demo/pull/5/commits/703fb9e403629747e28f6e93bb30f8a46990e1b9

Reply:
this is exactly what I meant!